frontend/src/scenes/funnels/FunnelViz.js

```diff
@@ -82,6 +84,11 @@ export function FunnelViz({
         }
     }, [stepsResult, funnelLoading])
 
+    const conversionWindowDebounced = useDebouncedCallback((value) => {
```

Comment:
Can you move this into the funnelLogic and use [kea breakpoints](https://kea.js.org/docs/guide/additional/#breakpoints) in the the listeners?